packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -477,10 +530,19 @@ class MediaQueryData {
 
   /// Creates a copy of this media query data but with the given fields replaced
   /// with the new values.
+  ///
+  /// If `textScaler` and `textScaleFactor` are both specified, `textScaleFactor`
+  /// will be ignored in favor of `textScaler`.
```

Comment:
Why not assert that both are not specified, like you did elsewhere?  Well I guess someone could migrate part of their code TextScaler before the rest?

Reply:
Big +1, here and elsewhere. Ignoring values silently could be a frustrating source of errors...